macros/LIGen.mac

```diff
@@ -0,0 +1,116 @@
+#Sample setup macro with no visualization
+
+/run/verbose 0
+/tracking/verbose 0
+/hits/verbose 0
+
+/WCSim/random/seed 3254
+
+# HyperK
+/WCSim/WCgeom HyperK_HybridmPMT_WithOD
+
+# HyperK Outer Detector
+/WCSim/HyperKOD/PMTODRadius 3inch
+/WCSim/HyperKOD/ODLateralWaterDepth 1 m
+/WCSim/HyperKOD/ODHeightWaterDepth 2 m
+/WCSim/HyperKOD/ODDeadSpace 600 mm
+/WCSim/HyperKOD/ODTyvekSheetThickness 1 mm
+/WCSim/HyperKOD/ODWLSPlatesThickness 6 mm
+/WCSim/HyperKOD/ODWLSPlatesLength 30 cm
+#/WCSim/HyperKOD/PMTODperCellHorizontal 3
+/WCSim/HyperKOD/PMTODperCellHorizontal 0
+/WCSim/HyperKOD/PMTODperCellVertical 0
+/WCSim/HyperKOD/PMTODPercentCoverage 0.22
+#/WCSim/HyperKOD/PMTODPercentCoverage 50
+/WCSim/HyperKOD/ODPMTShift 0 m
+#/WCSim/HyperKOD/DeactivateWLS
+
+# Update geom
+/WCSim/Construct # This must be uncommented in order to construct a new detector configuration.
+
+#Added for the PMT QE option 08/17/10 (XQ)
+# 1. Stacking only mean when the photon is generated
+# the QE is applied to reduce the total number of photons
+# 2. Stacking and sensitivity detector
+# In the stacking part, the maximum QE is applied to reduce
+# the total number of photons
+# On the detector side, the rest of QE are applied according to QE/QE_max
+# distribution. This option is in particular important for the WLS
+# 3. The third option means all the QE are applied at the detector
+# Good for the low energy running.
+# 4. Switch off the QE complety, ie. set it at 100%
+#/WCSim/PMTQEMethod     Stacking_Only
+#/WCSim/PMTQEMethod     Stacking_And_SensitiveDetector
+/WCSim/PMTQEMethod     SensitiveDetector_Only
+#/WCSim/PMTQEMethod     DoNotApplyQE
+
+#turn on or off the collection efficiency
+/WCSim/PMTCollEff on
+
+# command to choose save or not save the pi0 info 07/03/10 (XQ)
+/WCSim/SavePi0 false
+
+#choose the Trigger & Digitizer type (and options)
+/DAQ/Digitizer SKI
+#/DAQ/Trigger NDigits
+/DAQ/Trigger NoTrigger
+#/DAQ/RelativeHitTime true
+
+#grab the other DAQ options (thresholds, timing windows, etc.)
+/control/execute /data2/kneale/hk-software/wcsim_od_calib/macros/daq.mac
```

Comment:
General comment for this file - can you remove the personal paths? The preference is to assume you're running in `$WCSIMDIR` or `$WCSIM_BUILD_DIR`

Reply:
```suggestion
/control/execute macros/daq.mac
```
Think this was lost in the flurry of comments. This is the kind of thing I meant (there's a few places in the file)

Reply:
Argh sorry I'll fix these now.